src/main/java/com/faforever/client/map/MapPreviewLargeController.java

```diff
@@ -0,0 +1,70 @@
+package com.faforever.client.map;
+
+import com.faforever.client.game.MapInfoBean;
+import com.faforever.client.legacy.map.Comment;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.Region;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+
+import java.lang.invoke.MethodHandles;
+
+public class MapPreviewLargeController {
+
+  @FXML
+  ImageView largeImagePreview;
+  @FXML
+  Label mapNameLabel;
+  @FXML
+  Label maxPlayerLabel;
+  @FXML
+  Label mapSizeLabel;
+  @FXML
+  Label mapDescriptionLabel;
+  @FXML
+  Pane commentContainer;
+  @FXML
+  GridPane root;
+
+  private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
Static fields go before fields. You can also enable "Rearrange code" in your commit dialog to have such things fixed automatically.